pkg/virt-controller/watch/migration.go

```diff
@@ -1687,3 +1747,25 @@ func (c *MigrationController) isMigrationPolicyMatched(vmi *virtv1.VirtualMachin
 	migrationPolicyName := vmi.Status.MigrationState.MigrationPolicyName
 	return migrationPolicyName != nil && *migrationPolicyName != ""
 }
+
+func (c *MigrationController) isMigrationHandedOff(migrationKey string) bool {
+	c.handOffLock.Lock()
+	defer c.handOffLock.Unlock()
+
+	_, isHandedOff := c.handOffMap[migrationKey]
+	return isHandedOff
```

Comment:
you can't just use the map here. both the VMI status and map need to be checked. see my previous comment here, https://github.com/kubevirt/kubevirt/pull/7599#pullrequestreview-1045641863

Reply:
That's weird, I've already addressed these comments and [pushed the changes](https://github.com/kubevirt/kubevirt/compare/a1179db24a5a971ba9c84d789659d845588e2f1c..f24a456513d27d2c35411275747200295f3c49d8#diff-1baf192bb9c4560d7d40f008b7ba5c680ebce669b8f1b7639445abd3e7269688R1756).
Not sure what went wrong here.. Perhaps some git confusion..

I've pushed the changes again, it's ready now. Sorry for the noise.